configurations/manager/6TB_dataset.yaml

```diff
@@ -0,0 +1,31 @@
+test_duration: 1200
+round_robin: true
+
+prepare_write_cmd: [ "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1500000000",
+                     "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1500000001..3000000000",
+                     "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=3000000001..4500000000",
+                     "cassandra-stress write cl=ALL n=1500000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=650 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=4500000001..6000000000" ]
+
+stress_cmd: [ "cassandra-stress write cl=QUORUM duration=20m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=360 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..3000000000",
+              "cassandra-stress write cl=QUORUM duration=20m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=360 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=3000000001..6000000000", ]
+
+stress_read_cmd: [ "cassandra-stress read cl=QUORUM duration=20m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=360 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..3000000000",
+                   "cassandra-stress read cl=QUORUM duration=20m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=360 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=3000000001..6000000000", ]
+
+instance_type_db: 'i4i.8xlarge'
+instance_type_loader: 'c6i.8xlarge'
+n_db_nodes: 6
+n_loaders: 4
+n_monitor_nodes: 1
+region_name: us-east-1
+backup_bucket_location: 'manager-backup-tests-us-east-1'
+append_scylla_yaml:
+  auto_snapshot: false
+  object_storage_endpoints:
+    - name: s3.us-east-1.amazonaws.com
```

Comment:
@regevran isn't there default for those things ? i.e.  why a user need to configure cloud services urls ?

I can understand why for unit testing or something like that, one would want that, but why a user need to fuss around with all of this configuration ?

Reply:
quick call to:
```
curl http://169.254.169.254/latest/meta-data/placement/region -H "X-aws-ec2-metadata-token: $TOKEN"
```

and you should have the region of this node, which is exactly what 95% of user would want it to be